contrib/inventory/nodepool.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+'''
+Nodepool external inventory script
+==================================
+
+Generates an inventory that Ansible can understand by making Zookeeper calls
+to the Nodepool service using the kazoo library.
+Zookeeper IP and port are read from environment variables NODEPOOL_ZK_IP and
+NODEPOOL_ZK_PORT respectively. If those variables are not set, the script defaults
+to 127.0.0.1 for the address and 2181 for the port.
+Example:
+
+  #!/bin/bash
+  export NODEPOOL_ZK_IP=10.10.10.10
+  export NODEPOOL_ZK_PORT=9000
```

Comment:
It'd be super-nice if the standard nodepool yaml config file (/etc/nodepool/nodepool.yaml IIRC) could also be parsed by default, so if it's there, it gets used.